hostedzones/judiciary.uk.yaml

```diff
@@ -375,9 +375,17 @@ s2._domainkey.staging.hr:
 selector1._domainkey:
   type: CNAME
   value: selector1-judiciary-uk._domainkey.justiceuk.onmicrosoft.com.
+selector1-azurecomm-prod-net._domainkey:
+  type: CNAME
+  value: selector1-azurecomm-prod-net._domainkey.azurecomm.net
+  ttl: 3600
```

Comment:
`ttl` needs to come before `type`. This is the cause of the sync failed check.

Reply:
Thanks, will update